src/libraries/System.Private.Uri/src/System/Uri.cs

```diff
@@ -199,11 +198,9 @@ private bool IsNotAbsoluteUri
         //
         // Checks if Iri parsing is allowed by the syntax & by config
         //
-        private bool _iriParsing;
+        private bool IriParsing => IriParsingStatic(_syntax);
```

Comment:
I assume you validated that this has no impact on throughput?

Reply:
I saw no change in throughput after this change.